### DISCUSSION TITLE
Erro - Rebuilding ... tailwind installation

### DISCUSSION THREAD

**Discussion by MarcoScottiGitHub** [tailwindlabs/tailwindcss#14510] [Help]
Guys please, someone knows the reason for this error at the time I give this command : **npx tailwindcss -i ./src/input.css -o ./src/output.css --watch**

i do the step by step documentation , I have seen several videos on youtube , and even then I could not solve. could you help me out of kindness !

i await return!

thank you very much

![erro tailwind](https://github.com/user-attachments/assets/e5795420-f3c8-4f8a-9070-c040707ce362)
![erro](https://github.com/user-attachments/assets/26ed8fb9-9246-4722-9552-d4890c39bee9)
![erro2](https://github.com/user-attachments/assets/85ecdeda-ee49-4fd4-998c-6c663ec4e068)

**Comment by adamwathan** [comment]
Hey! Looks like your content configuration isn’t covering your source files. Can you share the contents of your Tailwind config file?

**Reply by MarcoScottiGitHub** [reply]
Yes friend .

would it be the **tailwind.config.js** configuration?

![congif](https://github.com/user-attachments/assets/d1cdd101-dc6e-4b90-8a47-1ecd80b1e3a2)

**Reply by adamwathan** [reply]
Can you share the contents of `index.html`?

**Reply by MarcoScottiGitHub** [reply]
yes yes,

![content html](https://github.com/user-attachments/assets/f5fa463e-6034-4f9c-afcf-7b2831d4b0e8)

**Comment by MarcoScottiGitHub** [comment]
inside SRC folder.

index.html / input.css / output.css
![src](https://github.com/user-attachments/assets/f25d37a8-4066-46b5-8dc6-a80fddf61a09)
